app/controllers/admin/merchants_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::MerchantsController < Admin::BaseController
+  def show
+    @merchant = Merchant.find(params[:id])
+  end
+end
```

Comment:
Should we use strong params here or are they not necessary since we're not updating the database?